pilot/cmd/pilot-agent/status/server.go

```diff
@@ -493,10 +493,16 @@ func (s *Server) handleStats(w http.ResponseWriter, r *http.Request) {
 	var envoyCancel, appCancel context.CancelFunc
 	defer func() {
 		if envoy != nil {
-			envoy.Close()
+			err = envoy.Close()
+			if err != nil {
+				log.Infof("envoy connection is not closed successfully with error : %s", err)
```

Comment:
```suggestion
				log.Infof("envoy connection is not closed successfully with error: %v", err)
```

Really nit but we shouldn't have spaces before `:` and `%s` doesn't really work with err I thought?